app/src/main/java/com/cornellappdev/transit/ui/components/LocationItem.kt

```diff
@@ -0,0 +1,86 @@
+package com.cornellappdev.transit.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.transit.R
+import com.cornellappdev.transit.ui.theme.TransitBlue
+
+@Composable
+fun LocationItem(image: Painter, editImage: Painter, label: String, sublabel: String, editing: Boolean){
+    Column(modifier =  Modifier.width(88.dp).height(86.dp).padding(horizontal = 8.dp)){
+        if(editing && label!="Add"){
+            Box(modifier = Modifier.size(48.dp).align(Alignment.CenterHorizontally)){
+                Image(
+                    editImage,
+                    contentDescription = "Place",
+                    modifier = Modifier
+                        .align(Alignment.Center)
+                        .size(48.dp)
+                )
+                Image(
+                    painter = painterResource(id = R.drawable.remove_icon),
+                    contentDescription = "Add",
+                    modifier = Modifier.align(Alignment.TopEnd).size(20.dp)
+                )
+
+            }
```

Comment:
It seems like we want to make this component clickable when it's an `Add` button.

There are a few things to consider with this in mind;

1. You'll need to conditionally add a `Modifier` here to the image component to make it `clickable`. You can look into the `.takeIf { ... }` modifier to do this.
2. You'll want to expose an onClick as a parameter to practice best UI component design (remember our onboarding?)

Reply:
Also--it looks like the images are a little blurry, no? I think you have to export them from Figma as some higher scale to get them not to blur:

![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/2a122d84-d00f-43a3-91a0-4fb9866fccd7)